docs/synapse/glossary.rst

```diff
@@ -98,21 +115,22 @@ Cell
 ----
 
 The Cell is a basic building block of Synapse services, including the Cortex. See :ref:`dev_architecture` for more
-information about what a Cell provides.
+information about the features provided by a Cell.
```

Comment:
Lets go with features provided by a "synapse-svc" reference. I'd like to avoid using the word Cell whenever possible. :)

Reply:
the 'dev-architecture' link doesn't refer to "Synapse services" at all so made it a generic "see link for additional detail".